torch/csrc/jit/tensorexpr/ir_simplifier.h

```diff
@@ -119,10 +119,10 @@ class Term : public ExprNode<Term> {
     sort();
   }
 
-  Term(HashProvider& hasher, const ExprPtr& s, const std::vector<ExprPtr>& v)
+  Term(HashProvider& hasher, ExprPtr s, std::vector<ExprPtr> v)
       : ExprNodeBase(promoteTypesVec(s, v)),
```

Comment:
Eh, are these safe with promotTypesVec, I don't think it's actually defined that the initializers run in order, what you had before was right.

Reply:
Ah sorry, I missed this line when reviewing the PR on my phone, what you had before was right.